### PR TITLE
feat: notifications now follow the theme

### PR DESCRIPTION
**Pull request by kevinjoshi46b**
Solves #286 

Notifications will look like this now:

Screenshot 2023-04-10 113745

Screenshot 2023-04-10 113736

Screenshot 2023-04-10 113801

Screenshot 2023-04-10 113809

### CODE REVIEW

File: packages/nextjs/utils/scaffold-eth/notification.tsx [hunk 1]
```diff
@@ -49,14 +49,14 @@ const Notification = ({
   return toast.custom(
     t => (
       <div
-        className={`flex flex-row items-center justify-between max-w-sm bg-white p-4 text-black shadow-2xl hover:shadow-none transform-gpu translate-y-0 hover:translate-y-1 rounded-md relative transition-all duration-500 ease-in-out space-x-2 ${
+        className={`flex flex-row items-start justify-between max-w-sm bg-base-200 p-4 shadow-2xl hover:shadow-none transform-gpu translate-y-0 hover:translate-y-1 rounded-md relative transition-all duration-500 ease-in-out space-x-2 ${
           t.visible ? "top-0" : "-top-96"
```
Comment (damianmarti) [inline packages/nextjs/utils/scaffold-eth/notification.tsx:53]: Why do you need the -top-96 when not visible?

Comment (kevinjoshi46b) [reply packages/nextjs/utils/scaffold-eth/notification.tsx:53]: I didn't notice it till you pointed it out.
I just checked, it seems that "-top-96" position helps create the slide up out of the screen animation, without it the notification just disappears in place but this brings up two points what about notifications with position "bottom-xxx"?
- In the case where notification position is bottom, when the notification is about to disappear it moves up to somewhere in the middle of the screen and seems to look like a glitch (the screenshot bellow explains it better) (will fix this in my next commit)

From here:
image

To here:
image

- When hovering on the notification the shadow is set to none and the notification moves down by one ("translate-y-1") this animation looks weird when notification position is bottom, I think it should have a mirror effect and the notification should move up by 1 ("translate-x-1"). Would love to have your opinion on this before I change it.

Comment (damianmarti) [reply packages/nextjs/utils/scaffold-eth/notification.tsx:53]: Maybe we can check how it feels if the notification is at the top.

Comment (kevinjoshi46b) [reply packages/nextjs/utils/scaffold-eth/notification.tsx:53]: I didn't get you there

Comment (damianmarti) [reply packages/nextjs/utils/scaffold-eth/notification.tsx:53]: Sorry. 

About "When hovering on the notification the shadow is set to none and the notification moves down by one ("translate-y-1") this animation looks weird when the notification position is bottom, I think it should have a mirror effect and the notification should move up by 1 ("translate-x-1"). Would love to have your opinion on this before I change it."

Maybe the animation looks good when the notification is at the top, so if you change it to "translate-x-1", now the notification at the top will look weird.

Please, let me know if I'm not clear enough.

Comment (kevinjoshi46b) [reply packages/nextjs/utils/scaffold-eth/notification.tsx]: Done

**Comment by technophile-04** [comment]
Thanks @kevinjoshi46b for the PR and @damianmarti for the review !!

> In the case where notification position is bottom, when the notification is about to disappear it moves up to somewhere in the middle of the screen and seems to look like a glitch (the screenshot bellow explains it better) (will fix this in my next commit)

Yeah this seemed to an issue, thanks for tackling it 

---


One minor change which I noticed was the position of the "X" icon : 

`main` branch : 
Screenshot 2023-04-12 at 8 15 41 PM

`this PR` : 
Screenshot 2023-04-12 at 8 13 47 PM

Was this intentional @kevinjoshi46b ? Both look fine to me but `main` branch seems a bit standardized with "X" at the top right not taking up space

--- 

Rest everything looks good to me, thanks @kevinjoshi46b 🙌

**Review by technophile-04 — APPROVED** [review]

**Comment by kevinjoshi46b** [comment]
> Was this intentional @kevinjoshi46b ? Both look fine to me but `main` branch seems a bit standardized with "X" at the top right not taking up space

I agree that position of "X" in the top right being standardized but I feel it looks like a standard only for models that have a lot of content in them.
What I have observed is for notifications that are meant to be just 1-2 lines having the "X" inline with the content looks more natural. That is also how [MUI](https://mui.com/material-ui/react-snackbar/) and many other libraries have implemented it.

But ya this is not that big of a deal if we want we can revert to the original positioning as well.

**Comment by kevinjoshi46b** [comment]
> 💯 agree, I just mentioned this because I noticed it while testing for long error messages because lol sometimes we get really big error messages check #301 but current looks good to me

Ya I am looking forward to that issue being resolved!
Those json notifications might get too overwhelming for non-dev users 🫠

**Pull request merged**